GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,13 +8,145 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
```

Comment:
This works but instead of doing a long switch statement you can just use the allSeasons array of arrays I mentioned above and just do 

`return allSeasons[section + 1].count`